ansible/azure-nat-gateway.yaml

```diff
@@ -0,0 +1,89 @@
+---
+
+- name: Configure Azure NAT Gateway
+  become: false
+  connection: local
+  hosts: localhost
+  gather_facts: false
+  vars:
+    kubeconfig: "{{ lookup('env', 'KUBECONFIG') }}"
+    resource_prefix: "coco"
+  tasks:
+    - name: Get Azure credentials
+      kubernetes.core.k8s_info:
+        kind: Secret
+        namespace: openshift-cloud-controller-manager
+        name: azure-cloud-credentials
+      register: azure_credentials
+      retries: 20
+      delay: 5
```

Comment:
Should we just make this a safe exit? if we are not on azure in the future a clean exit would be nice.

Reply:
I had been thinking about it, but we should holistically do it across the board 

such as: https://github.com/sabre1041/coco-pattern/blob/main/ansible/configure-issuer.yaml

Reply:
Actually I agree. I'll raise an issue to chart the development of the feature as we discussed offline.